README.md

```diff
@@ -19,18 +20,106 @@ Coupler Specific Shared Development
+# Running AMIP
+Here we will focus on the AMIP experiment, which uses the environment in the `experiments/ClimaEarth/` subdirectory of ClimaCoupler.jl
+The first step to do this is to install all required packages for the environment using the following Julia command:
+```julia
+julia --project=experiments/ClimaEarth -E "using Pkg; Pkg.instantiate(); Pkg.build()"
+```
+
+Now you're ready to run the experiment, which uses the `run_amip.jl` driver. To run interactively:
+```
+julia --project=experiments/ClimaEarth
+julia> include("run_amip.jl")
+```
+
+Or to run from the terminal:
+```julia
+julia --project=experiments/ClimaEarth run_amip.jl
+```
+
+When running from the terminal, you can also specify a configuration file to use for the simulation setup, and a job ID to keep track of this run's output.
+Existing configuration files are specified in the `config/` directory within ClimaCoupler.jl.
+For example, to run a coarse AMIP run using Float64, you could use the following command:
+```julia
+julia --project=experiments/ClimaEarth run_amip.jl --config_file config/ci_configs/coarse_single_ft64.jl --job_id coarse_single_ft64
+```
+
+Output from your run will be saved in the folder `experiments/ClimaEarth/output/amip/<job_id>/` for AMIP runs, or
+`experiments/ClimaEarth/output/slabplanet/<job_id>/` for slabplanet runs. If no configuration file is specified, the default
+`interactive_debug.yml` will be used, and output will be saved in `experiments/ClimaEarth/output/slabplanet/interactive_debug/`.
+
+> Note: If you want to set the configuration file to something other than the default
+while running the driver interactively, you'll need to
+manually set the values for `parsed_args["config_file"]` and `parsed_args["job_id"]`.
+
+### A Note about ClimaComms and MPI
+If you don't intend to run your simulation using MPI, but you see an error about MPI and your simulation crashes,
+ClimaComms may be incorrectly selecting the configuration for your run.
+In this case, you can force ClimaComms to ignore MPI with
+```
+export CLIMACOMMS_CONTEXT="SINGLETON"
+```
+from the terminal, or
+```
+ENV["CLIMACOMMS_CONTEXT"]="SINGLETON"
+```
+from within the Julia environment before running the experiment
+
+## Running on GPU or with MPI
+
+### CUDA.jl and MPI.jl packages
+The CUDA.jl and MPI.jl are required to run ClimaCoupler's AMIP experiment on GPU and in parallel (with MPI), respectively.
+Not every machine is capable of running on GPU or with MPI, and the AMIP experiment can be run on CPU
+without these packages, so they aren't included in the ClimaCoupler AMIP experiment environment.
+This means that if you want to run our driver using these capabilities (locally or remotely), you should install
+CUDA and MPI in your machine's base Julia environment. This will make the packages available to the
+AMIP experiment environment, as it is a sub-environment of the base environment.
```

Comment:
Another option is to install them in your Project.toml for ClimaEarth (if you don't want to have them in your base environment)

Reply:
I thought we wanted to avoid having them in the ClimaEarth environment for the same reasons we exclude them from the top-level ClimaCoupler environment

Reply:
We (developers) don't add it, but users are free to modify an enviroment on your machine if they want to. In practice, at this point, everyone is both a developer and a user, so we wouldn't do it. But if you are taking the perspective of a pure user, there's no problem with modifying their project.